scripts/image-builder/build.sh

```diff
@@ -246,8 +251,22 @@ if [ ! -z ${CUSTOM_RPM_FILES} ] ; then
     createrepo custom-rpms >/dev/null
 fi
 
+# Install prometheus process exporter
+rm -rf prometheus-rpms 2>/dev/null || true
+if ${PROMETHEUS} ; then
+  owner=ncabatoff
+  repo=process-exporter
+  version=$(curl -s https://api.github.com/repos/${owner}/${repo}/releases/latest | jq -r '.name')
+  url=https://github.com/${owner}/${repo}/releases/download/${version}/
+  file=${repo}_${version#v}_linux_amd64.rpm
+
+  wget -q -P prometheus-rpms ${url}${file}
+  CUSTOM_RPM_FILES+="$(pwd)/prometheus-rpms/${file},"
```

Comment:
This is a bit inconsistent. We should either create a new repo and leave custom RPMs alone, or we should download to custom-rpms directory and not create a new repo.

Reply:
Custom RPMs has an existing mechanism to modify the blueprint and I'm reusing that rather than duplicating it.

Reply:
OK, if we want to reuse custom RPMs, let's move the code of changing CUSTOM_RPM_FILES setting before the line 244. This way, we should not need prometheus-rpms.toml template at all - all the prometheus RPMs will be copied to custom-rpms folder and included automatically.